client/blocks/signup-form/crowdsignal.scss

```diff
@@ -174,9 +171,20 @@ body.is-section-signup .layout .signup.is-crowdsignal .step-wrapper {
 	}
 }
 
-.button.signup-form__crowdsignal-wpcom {
+.components-button.signup-form__crowdsignal-wpcom {
 	display: flex;
+	height: 44px;
 	margin-bottom: 40px;
+	padding-inline: 16px;
+
+	> svg {
+		margin-right: auto;
+	}
+
+	> span {
+		margin-left: -20px;
+		margin-right: auto;
+	}
```

Comment:
Has the WordPress logo on the left, so need to match the layout of the social buttons below